### ISSUE TITLE
Error with Capacitor v3 

### ISSUE THREAD

**Issue by GosuMania** [capacitor-community/native-market#10]
If I use this plugin with capacitor v3, I get this error:

`Error: node_modules/@capacitor/core/types/definitions-internal.d.ts:16:18 - error TS2430: Interface 'CapacitorInstance' incorrectly extends interface 'CapacitorGlobal'.
Types of property 'Plugins' are incompatible.
16 Property 'NativeMarket' is missing in type '{ [pluginName: string]: { [prop: string]: any; }; }' but required in type 'PluginRegistry'.

export interface CapacitorInstance extends CapacitorGlobal {
~~~~~~~~~~~~~~~~~

node_modules/@capacitor-community/native-market/dist/esm/definitions.d.ts:3:9
3 NativeMarket: NativeMarketPlugin;
~~~~~~~~~~~~
'NativeMarket' is declared here.`

**Comment by tfathy** [comment]
I havr the same error. any suggestions ?

**Comment by hermitdemschoenenleben** [comment]
this fork works for me with capacitor 3: https://github.com/djonmayer/native-market

**Comment by riderx** [comment]
if you need i published it with capacitor 3 https://www.npmjs.com/package/@capgo/native-market

**Comment by jcesarmobile** [comment]
I've released 4.0.0 with Capacitor 4 support, it should have fixed this issue.
Also 5.0.0 with Capacitor 5 support.
If the problem is still present, please, create a new issue and provide a sample app that reproduces the issue

There won't be a release with Capacitor 3 support.
